packages/trpc/server/routers/viewer/teams/removeMember.handler.ts

```diff
@@ -14,11 +15,16 @@ type RemoveMemberOptions = {
   ctx: {
     user: NonNullable<TrpcSessionUser>;
     prisma: PrismaClient;
+    sourceIp?: string;
```

Comment:
This is always there - check getIP return signature. Type issue within context i cant seem to figure out